server/context_table.md

```diff
@@ -0,0 +1,77 @@
+# Context Table
```

Comment:
Better place for this file?

Reply:
Sounds like a good place for now, thanks, some [suggested fixes](https://github.com/openmsupply/open-msupply/blob/2a3d571f4a6d1086e3356098595da5b8e8f08563/server/context_table.md)